RiotSharp/StaticRiotApi.cs

```diff
@@ -1,4 +1,6 @@
 ﻿using Newtonsoft.Json;
+using RiotSharp.Http;
+using RiotSharp.Interfaces;
 using RiotSharp.StaticDataEndpoint;
 using System;
 using System.Collections.Generic;
```

Comment:
Oh that slipped by. Probably didn't throw a build error when I changed the namespaces.